markdown_lab/core/scraper.py

```
@@ -502,8 +494,10 @@ def _convert_content(
             # Then convert to the requested format
             try:
                 # Try to use functions from markdown_lab_rs for conversion
-                from markdown_lab.markdown_lab_rs import (document_to_xml,
-                                         parse_markdown_to_document)
+                from markdown_lab.markdown_lab_rs import (
+                    document_to_xml,
+                    parse_markdown_to_document,
+                )
```

Pylint (3.3.7)

[convention] Import outside toplevel (markdown_lab.markdown_lab_rs.document_to_xml, markdown_lab.markdown_lab_rs.parse_markdown_to_document)

(C0415)